By the comment's own admission, this cannot affect on-disk nor ABI because
'long' varies per architecture.  So, this change is always correct.

--- src/dbinc/clock.h	2013-09-09 15:35:08.000000000 +0000
+++ src/dbinc/clock.h	2020-01-31 19:22:11.630000000 +0000
@@ -54,11 +54,7 @@
  */
 typedef struct {
 	time_t	tv_sec;				/* seconds */
-#ifdef HAVE_MIXED_SIZE_ADDRESSING
-	int32_t tv_nsec;
-#else
-	long	tv_nsec;			/* nanoseconds */
-#endif
+	time_t	tv_nsec;			/* nanoseconds */
 } db_timespec;
 
 /* Operations on timespecs */
